contracts/Aavegotchi/facets/BridgeFacet.sol

```diff
@@ -43,11 +45,10 @@ contract BridgeFacet is Modifiers {
             LibItems.removeFromOwner(owner, id, value);
             LibERC1155Marketplace.updateERC1155Listing(address(this), id, owner);
         }
-        emit LibERC1155.TransferBatch(owner, owner, address(0), _ids, _values);
+        IEventHandlerFacet(s.wearableDiamond).emitTransferBatchEvent(owner, owner, address(0), _ids, _values);
```

Comment:
Is it intended to remove the emitted transfer event here? I think it'd be appropriate for both the aavegotchi diamond and the periphery to emit the events to keep compatibility with any integrations tracking events from the main diamond.

Reply:
yes...having the same event emitted twice in the same call frame is not appropriate....it can be left in if wanted but it means that marketplaces that track aavegotchi wearables will still use the main diamond instead of switching

Reply:
> yes...having the same event emitted twice in the same call frame is not appropriate....it can be left in if wanted but it means that marketplaces that track aavegotchi wearables will still use the main diamond instead of switching

Marketplaces shouldn't be using the AavegotchiDiamond anymore for ERC1155 because we're going to remove the ERC165 interface. Maybe there's a case for emitting both events?

Reply:
Not that I know of

Reply:
> yes...having the same event emitted twice in the same call frame is not appropriate....it can be left in if wanted but it means that marketplaces that track aavegotchi wearables will still use the main diamond instead of switching

If you don't continue emitting the events in the AavegotchiDiamond for items/wearables then any existing software that is using those events will break.  Do we know what trouble that will cause? 

Any existing third-party applications may not appreciate having their software suddenly not working anymore and may not appreciate being forced to make changes.

In addition software that breaks because of this change could possibly lead to users making mistakes or having trouble because the software they are using isn't working correctly anymore.   

Of course if there are no third-party applications using the existing item/wearable events then that is not a problem. Do we know?

Will all Pixelcraft software or user interfaces be updated to use the new events and so not be broken by this change?